plugins/janus_nosip.c

```diff
@@ -1798,107 +1874,86 @@ static int janus_nosip_allocate_local_ports(janus_nosip_session *session, gboole
 			close(session->media.pipefd[1]);
 			session->media.pipefd[1] = -1;
 		}
+
+		static unsigned session_count = 0;
+		static int fds[DEFAULT_RTP_RANGE_MAX - DEFAULT_RTP_RANGE_MIN + 1] = {0, };
+		if(session_count == 0) {
+			unsigned n_ports_locked = 0;
+			for(uint16_t port = rtp_range_min; port < rtp_range_max - 4; port++) {
+				int fd = socket(AF_INET, SOCK_DGRAM, 0);
+				if(fd == -1) {
+					// https://askubuntu.com/questions/162229/how-do-i-increase-the-open-files-limit-for-a-non-root-user
```

Comment:
Please remove that comment.

Reply:
The second commit is not for merging, just for testing.